src/components/CloudLogin/CloudLogin.tsx

```diff
@@ -76,15 +104,28 @@ export const CloudLogin: React.FC<CloudLoginProps> = ({
             gap: 0,
           }}
         >
-          <Flex gap="6px">
-            <Radio value="free" checked={selected === "free"} />
+          <Flex align="center">
+            <Radio mr="2" value="free" checked={selected === "free"} />
             <Text size="3">Free plan</Text>
           </Flex>
-          <Text size="2">- Code completions: Refact 1.6 model</Text>
-          <Text size="2">- In-IDE Chat: GPT-4o mini</Text>
-          <Text size="2">- Toolbox (refactor code, find bugs, etc.)</Text>
-          <Text size="2">- 2048-context length for completions</Text>
-          <Text size="2">- 8k context length for chat</Text>
+          <Flex pl="5" direction="column">
+            <Text size="2">
+              <i style={bulletStyle}></i>Code completions: Refact 1.6 model
```

Comment:
would the `Quote` component work here ?

Reply:
We could make a Quote component with a custom style like `src/Components/Text` that should get rid of the linter warnings about exporting too many components.